CONTRIBUTING.de.md

```diff
@@ -64,30 +64,30 @@ GitHub hat eine Integrierte Unterstützung für AGC assembly. Dasselbe gilt für
 
 ## Formatierung
 
-**Anmerkung:** GitHub und die oben erwähnten Erweiterungen werden automatisch sicherstellen dass sie die Korrekte Formatierung einhalten.
+**Anmerkung:** GitHub und die oben erwähnten Erweiterungen werden automatisch sicherstellen, dass du die korrekte Formatierung einhältst.
 
 - benutze tab indentation
 - benutze tab width von 8
-- entferne nachlaufende leerstellen
+- entferne Leerzeichen am Ende einer Zeile
 
-## Was sol ich überprüfen  ?
+## Was soll ich überprüfen?
 
-Jegliche Ungereimtheiten zwischen den scans und den Code in die repository, einschließlich:
+Alle Ungereimtheiten zwischen den Scans und dem Code in diesem Repository, einschließlich:
 
 ### Kommentare
 
-- Kommentare in dem Abgeschriebenen Code müssen denen in den scans gleichen
-  - Das könnte extra einen sprachlichen Fehler oder das entfernen/hinzufügen eines ganzen Kommentars bedeuten
+- Kommentare in dem abgeschriebenen Code müssen denen in den Scans gleichen
+  - Das kann bedeuten, dass man einen Rechtschreibfehler übernehmen oder einen ganzen Kommentar entfernen/hinzufügen muss
 
 ### Zeilenumbruch
 
-- Zeilenumbruch *mit* `R0000` in Spalte 1 sollte genau mit den scans übereinstimmen
-- Zeilenumbruch *mit**__out__* `R0000` in Spalte 1 sollte nur 1 bis zwei Leerzeilen nacheinander haben
-  - Wenn dort mher als 2 leere Zeilenumbrüche sind sollen diese entfernt werden
+- Zeilenumbrüche *mit* `R0000` in Spalte 1 sollte genau mit den Scans übereinstimmen
+- Zeilenumbrüche *__ohne__* `R0000` in Spalte 1 sollten nur ein bis zwei Leerzeilen nacheinander haben
+  - Wenn es dort mehr als zwei Leerzeilen gibt, sollten die zusätzlichen Leerzeilen entfernt werden.
     - Zeilen mit `R0000` in Spalte 1 zählen nicht dazu
-  - In den Quell Bildern, Diese wurden verursacht durch eine nicht gedruckte stelle in in Spalte 8. A 2 dort hat eine doppelte Leerstelle (einezilne Leerzeile) und A 3 hat eine dreifache Leerstelle (doppelte Leerzeile). Werte 4-8 wurden definiert aber nie benutzt. Mehr hier zu in [#159][7]
+  - In den Originalbildern wurden die verursacht durch eine nicht gedruckte Zahl in in Spalte 8. Eine 2 dort hat eine doppelte Leerstelle (einzelne Leerzeile) und eine 3 hat eine dreifache Leerstelle (doppelte Leerzeile). Die Werte 4-8 wurden definiert, aber nie benutzt. Mehr dazu hier: [#159][7]
```

Comment:
Possibly "Leerzeichen" (more commonly used translation for "space") instead of "Leerstelle"?

Reply:
I was not sure about this one. Since it's related to the original scans and therefore related to actual paper, I thought that "Leerstelle" would be fine. But I can change that as well.